packages/core/package.json

```diff
@@ -23,7 +23,8 @@
   "author": "IBM",
   "license": "Apache-2.0",
   "dependencies": {
-    "resize-observer-polyfill": "1.5.0"
+	"resize-observer-polyfill": "1.5.0",
```

Comment:
Has this also been taken out of the bundle?

Reply:
not that I'm aware ... I haven't touched any of the package.jsons, so that's how it was before my changes